ui-admin/src/portal/siteContent/SiteContentEditor.tsx

```diff
@@ -2,15 +2,16 @@ import React, { useState } from 'react'
 import { NavbarItemInternal, PortalEnvironment } from 'api/api'
 import Select from 'react-select'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faPlus } from '@fortawesome/free-solid-svg-icons'
+import { faClockRotateLeft, faPlus } from '@fortawesome/free-solid-svg-icons'
 import HtmlPageEditView from './HtmlPageEditView'
 import { HtmlPage, LocalSiteContent, ApiProvider, SiteContent, ApiContextT } from '@juniper/ui-core'
 import { Link } from 'react-router-dom'
 import SiteContentVersionSelector from './SiteContentVersionSelector'
 import { Button } from '../../components/forms/Button'
+import AddPageModal from './AddPageModal'
 
-type NavbarOption = {label: string, value: string | null}
-const landingPageOption = { label: 'Landing page', value: null }
+type NavbarOption = {label: string, value: string}
```

Comment:
Previously the `value` field of `NavbarOption` was the ID of the `NavbarItem`. Landing page was the null/special case because it wasn't associated with a navbar item. I had to switch `value` to be the page title/text of the navbar item because newly added pages don't have an ID until they've been saved to the database. So the special-case for Landing page was no longer unique because I don't want to force a save just to make the page dropdown work correctly. My assumption is that navbar item text is unique _enough_ to not really be a concern, but maybe it'd be worth protecting against that?

Reply:
yeah, I can't imagine people wanting nav links with duplicate text